.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,14 +1,8 @@
 blank_issues_enabled: true
 contact_links:
-  - name: Jira
-    url: https://issues.redhat.com/projects/KOGITO
-    about: Kogito Jira Board
+  - name: GitHub Issues
+    url: https://github.com/kiegroup/kogito-serverless-operator/issues
```

Comment:
I thought we would be using https://github.com/kiegroup/kie-issues

Reply:
We can change that later.

Reply:
ok